playbooks/roles/setup_pipeline/tasks/main.yml

```diff
@@ -0,0 +1,137 @@
+---
+- name: create application user
+  user:
+    name: "{{ pipeline_user }}"
+    home: "{{ pipeline_app_dir }}"
+    createhome: no
+    shell: /bin/bash
+    generate_ssh_key: yes
+  tags:
+    - install
+    - install:base
+
+- name: create edxapp user dirs
+  file:
+    path: "{{ item.path }}"
+    state: directory
+    owner: "{{ pipeline_user }}"
+    group: "{{ common_web_group }}"
+    mode: "{{ item.mode | default(0755) }}"
+  with_items:
+    - { path: "{{ pipeline_app_dir }}" }
+    # needed for the ansible 1.5 git module
+    - { path: "{{ pipeline_app_dir }}/.ssh" }
+    - { path: "{{ pipeline_hadoop_tasks_dir }}" }
+    - { path: "{{ pipeline_venv_dir }}" }
+  tags:
+    - install
+    - install:base
+
+- name: own key authorized
+  file:
+    src: "{{ pipeline_app_dir }}/.ssh/id_rsa.pub"
+    dest: "{{ pipeline_app_dir }}/.ssh/authorized_keys"
+    owner: "{{ pipeline_user }}"
+    group: "{{ pipeline_group }}"
+    force: yes
+    state: link
+
+- name: ensure user is in sudoers
```

Comment:
@morgan no, we need sudo perms to run the `remote-task`, I don't fully understand the command yet, but is a wrapper to run the scheduled hadoop tasks.